apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/SyntheaDataToFhirIT.java

```diff
@@ -138,22 +138,26 @@ public void checkThatSyntheaDataDoesntGoBoom() throws FHIRException {
      * there 100% of the time.
      */
     Assert.assertTrue(
-        "No diagnoses found.", eobs.stream().anyMatch(eob -> !eob.getDiagnosis().isEmpty()));
-    Assert.assertTrue(
-        "No procedures found.", eobs.stream().anyMatch(eob -> !eob.getProcedure().isEmpty()));
+        "No diagnoses or procedures found.",
+        eobs.stream()
+            .anyMatch(eob -> !(eob.getDiagnosis().isEmpty() && eob.getProcedure().isEmpty())));
     Assert.assertTrue(
         "No line item services found.",
         eobs.stream()
             .flatMap(eob -> eob.getItem().stream())
             .anyMatch(item -> item.getService() != null));
+    // These tests are commented out pending discussions on how to proceed with synthetic vs
+    // real NPIs.
+    /*
```

Comment:
TBD, but if we do decide to not support lookups for the synthetic NPIs, we'll want to remove these assertions.

Reply:
This was reverted, since we've decided to pull the NPI data changes out.